src/core/common/uri_facade_win.cpp

```diff
@@ -19,7 +19,7 @@
 namespace Common
 {
 
-void Uri::Initialize(const char * uriString, std::size_t size)
+void Uri::Initialize(const std::string &uriString)
```

Comment:
This changes the signature of the method. Can we avoid this?

Reply:
Note that it is specifically changed to match [the corresponding declaration in the header file](https://github.com/FreeOpcUa/freeopcua/blob/89cf09977ad80fb785e64d0dcf1d68c5351fc54f/include/opc/common/uri_facade.h#L21).

[In the Linux version, the signature is already correct.](https://github.com/FreeOpcUa/freeopcua/blob/89cf09977ad80fb785e64d0dcf1d68c5351fc54f/src/core/common/uri_facade_lin.cpp#L21)

Reply:
cool!